src/Evolution/Systems/ScalarTensor/StressEnergy.hpp

```diff
@@ -3,4 +3,93 @@
 
 #pragma once
 
-namespace ScalarTensor {}  // namespace ScalarTensor
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "Evolution/Systems/ScalarTensor/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace ScalarTensor {
+
+/*!
+ * \brief Add in the trace-reversed stress-energy source term to the \f$\Pi\f$
+ * evolved variable of the ::gh system.
+ *
+ * \details The only stress energy source term in the Generalized Harmonic
+ * evolution equations is in the equation for \f$\Pi_{a b}\f$:
+ * \f[
+ * \partial_t \Pi_{ab} + \text{\{spatial derivative terms\}} =
+ * \text{\{GH source terms\}}
+ * - 16 \pi \alpha (T^{(\Psi)}_{ab} - \frac{1}{2} g_{a b} g^{cd}T^{(\Psi)}_{cd})
+ * ~.
+ * \f]
+ *
+ * (note that this function takes as argument the trace-reversed stress-energy
+ * tensor)
+ *
+ * This function adds that contribution to the existing value of `dt_pi`. The
+ * spacetime terms in the GH equation should be computed before passing the
+ * `dt_pi` to this function for updating.
+ *
+ * \see `gh::TimeDerivative` for details about the spacetime
+ * part of the time derivative calculation.
+ */
+void add_stress_energy_term_to_dt_pi(
+    gsl::not_null<tnsr::aa<DataVector, 3_st>*> dt_pi,
+    const tnsr::aa<DataVector, 3_st>& trace_reversed_stress_energy,
+    const Scalar<DataVector>& lapse);
+
+/*!
+ * \brief Compute the trace-reversed stress-energy tensor of the scalar field.
+ *
+ * \details The trace-reversed stress energy tensor is needed to compute the
+ * backreaction of the scalar to the spacetime evolution and is given by
+ * \f{align*}{
+ * T^{(\Psi), \text{TR}}_{a b} &\equiv T^{(\Psi)}_{ab} - \frac{1}{2}
+ *  g_{a b} g^{cd}T^{(\Psi)}_{cd} \\
+ *  &= \partial_a \Psi \partial_b \Psi ~,
+ * \f}
+ *
+ * where \f$T^{(\Psi)}_{ab}\f$ is the standard stress-energy tensor of the
+ * scalar.
+ *
+ * In terms of the evolved variables of the scalar,
+ * \f{align*}{
+    T^{(\Psi), \text{TR}}_{00} &= \alpha^2 \Pi^2 ~, \\
+    T^{(\Psi), \text{TR}}_{j 0} &= T^{(\Psi), \text{TR}}_{0j}
+                                 = - \alpha \Pi \Phi_j ~, \\
+    T^{(\Psi), \text{TR}}_{ij} &= \Phi_i \Phi_j ~,
```

Comment:
Yes, I understand this might be somewhat confusing. I thought about adding a superindex ^(\Psi) or sth, but felt it was too much notation. I listed the evolved variables in `System` with their names. Maybe could write a warning to not assume contractions of them.